backend/adminRouter.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const Students = require('../models/students');
+
+const adminRouter = express.Router();
+
+dishRouter.use(bodyParser.json());
+
+adminRouter.route('/')
+/*.get((req,res,next) => {
+    Students.find({})
+    .then((students) => {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(students);
+    }, (err) => next(err))
+    .catch((err) => next(err));
+})*/
+.post((req, res, next) => {
+    Students.create(req.body)
+    .then((student) => {
+        console.log('Student record created', student);
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(student);
+    }, (err) => next(err))
+    .catch((err) => next(err));
+});
+.delete((req, res, next) => {
+    Students.remove(req.body)
+    
+    }, (err) => next(err))
+    .catch((err) => next(err));
+});
+
+
+
+adminRouter.route('/:studentId')
+.get((req,res,next) => {
+    Students.findById(req.params.studentId)
+    .then((student) => {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.json(student);
+    }, (err) => next(err))
+    .catch((err) => next(err));
+})
+.post((req, res, next) => {
```

Comment:
I think we do not need this part. Please explain if there are any reasons.

Reply:
I thought the get method with the studentId might be necessary if a specific student record is asked for instead of all the records